src/scss/App.scss

```diff
@@ -0,0 +1,184 @@
+$background-color-main: #F4F4F4;
+$background-color-purple: #D0073E;
+$background-color-grey: #EDEDED;
+$text-dark-grey: #919191;
+$border-shadow: #D4D3D3;
+$border-dark-grey: #919191;
+$grey-three: #919191;
+
+@font-face {
+  src: url('../assets/fonts/Abadi-MT-Condensed-Extra-Bold.ttf');
+  font-family: Abadi;
+}
+
+@font-face {
+  src: url('../assets/fonts/AvenirNextLTPro-Cn.otf');
+  font-family: Avenir;
+}
+
+@font-face {
+  src: url('../assets/fonts/nba-spurs.ttf');
+  font-family: spurs;
+}
+
+body {
+  background-color: $background-color-main;
+  margin: 0;
+}
+
+.app {
+  text-align: center;
+}
+
+.active {
+  color: $background-color-purple !important;
+  border-bottom: 4px solid $background-color-purple;
+}
+
+.players {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-evenly;
+  list-style: none;
+  font-family: Abadi;
+  height: 45px;
+  box-shadow: 1px -1px 2px #D4D3D3;
+
+  .team {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    position: relative;
+    background: $background-color-purple;
+
+    .logo-container {
+      display: flex;
+      background-color: white;
+      border-radius: 24px;
+      box-shadow: 1px 1px 5px $border-dark-grey;
+      margin: 0 15px;
+    }
+
+    button {
+      background: none;
+      color: inherit;
+      border: none;
+      padding: 0;
+      cursor: pointer;
+      outline: inherit;
+      color: white;
+      text-transform: uppercase;
+      font-size: 22px;
+      font-family: spurs;
+      margin: 0 15px 0 0;
+
+      i {
+        display: inline-block;
+        vertical-align: middle;
+        margin-left: 10px;
+      }
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      left: 100%;
+      z-index: 99;
+      border-top: 45px solid #D0073E;
+      border-right: 20px solid transparent;
+    }
+  }
+
+  .links {
+    position: relative;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex: 1.5;
+    background-color: white;
+    font-family: Avenir;
+    text-transform: uppercase;
+    font-size: 16px;
+  
+    .link-container {
+      display: inline-block;
+      height: 100%;
+      margin: 0 25px;
```

Comment:
Change this `margin: 0 25px` to `padding: 0 25px` just so the user can hover over the the edge of the letters and it will be a smoother UX